### DISCUSSION TITLE
Phase field fracture (simple and cohesive zone model)

### DISCUSSION THREAD

**Discussion by saad493** [idaholab/moose#22024] [Q&A Modules: Phase field] [answered]
Can anyone guide where will the easy tutorials be found for learning as a beginner? I have already gone through every phase field theory and examples available on moose platform. I am finding difficulty in writing my own personal input file with imported mesh. Are there any handbook or step by step guidelines available for writing moose code?

**Comment by hugary1995** [comment, marked as answer]
1. https://mooseframework.inl.gov/syntax/Mesh/ talks about the mesh system.
2. Phase field fracture examples can be found in `modules/combined/test/tests/phase_field_fracture`.
3. There is an external app called RACCOON that specializes in phase field fracture. AT1, AT2, PFCZM, etc.
4. DG-like CZM examples are in `modules/tensor_mechanics/test/tests/cohesive_zone_model`

**Reply by MissCat-2025** [reply]
You really helped me a lot

**Reply by MissCat-2025** [reply]
Especially the third point

**Reply by MissCat-2025** [reply]
thank you
